### PR TITLE
[godot3] Move vrm load logic into vrm_loader.gd, add example of loading model at runtime

### PR DESCRIPTION
**Pull request by you-win**
Tested by:
- Created a release export template using `scons platform=windows tools=no target=release -j4` and tested with the included `vrm_samples/load_at_runtime_scene.tscn` scene.
- Opened `.vrm` files in the editor after toggling the plugin off and on

### CODE REVIEW

File: vrm_samples/load_at_runtime_scene.gd [hunk 1]
```diff
@@ -0,0 +1,9 @@
+extends Spatial
+
+func _ready():
+	var vrm_loader = load("res://addons/vrm/vrm_loader.gd").new()
+	
+	var model: Spatial = vrm_loader.import_scene("res://vrm_samples/AliciaSolid_vrm-0.51.vrm", 1, 1000)
```
Comment (fire) [inline vrm_samples/load_at_runtime_scene.gd:6]: Why does this load alicia?

Comment (you-win) [reply vrm_samples/load_at_runtime_scene.gd:6]: Alicia is provided as an example model already. In order to illustrate that models can be loaded via code at runtime, the Alicia model is loaded.

**Review by fire — COMMENTED** [review]
Is it possible to use `git mv` so that git understands the files are the same?

**Comment by you-win** [comment]
The files are not the same. `import_vrm.gd` has been stripped down to just be an editor plugin. That old functionality was moved to `vrm_loader.gd`

**Comment by fire** [comment]
Things have changed plenty. Is this still valid?

**Comment by you-win** [comment]
@fire Rebased off of the godot3 branch

**Comment by fire** [comment]
I am interested, do you think this can also be ported to master?

**Comment by lyuma** [comment]
This feature was already merged in master on January 5th, along with the update to the new importer and GLTF apis.

fire seems to think it will also work fine in 3.x, so I'll look into merging it into 3.x soon. Sorry for the delay (I was concerned with runtime support of PackedSceneGLTF.)

**Comment by you-win** [comment]
Thanks for taking another look at this PR!

**Comment by fire** [comment]
@you-win I don't see any activity and we're mostly in the Godot 4 branch.

**Comment by fire** [comment]
I am going to close this for now. I don't know where Godot Engine 3 code is anymore.

**Comment by you-win** [comment]
@fire There was no activity since this PR has been ready to merge for a while

**Comment by lyuma** [comment]
I never properly tested runtime import in Godot 3, and I had concerns about officially supporting this functionality since GLTF APIs are generally not exposed at runtime in Godot 3. (We do technically have the GDNative loader which may work at runtime, but I haven't tested it recently).

Keeping it as an open PR will enable users to find it. It is a useful change for some users, so I'd rather keep it open for now and if I do any maintenance work on Godot 3, I'll take another look

**Comment by fire** [comment]
Will close since there's not much movement and Godot 4.0 has been out for a few years.

**Pull request closed without merging**